jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/JavaLayerConfigurationsHelperTest.java

```diff
@@ -25,17 +25,23 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 import java.util.function.Function;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
 import org.junit.Assert;
+import org.junit.ClassRule;
 import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
 
 /** Tests for {@link JavaLayerConfigurationsHelper}. */
 public class JavaLayerConfigurationsHelperTest {
 
+  @ClassRule public static TemporaryFolder temporaryFolder = new TemporaryFolder();
```

Comment:
Since this is only one test, this would work, but if we were to add more tests this `TemporaryFolder` would be shared between the tests. This is okay if we have our convention be that we would never use the `TemporaryFolder` root for a test and always call `newFolder`, but in other parts of the codebase we have `TemporaryFolder` be non-static and we do use the root sometimes. Doesn't have to be in this PR, but I think we should just set some convention here and make sure we are consistent across all our uses of `TemporaryFolder`.